### ISSUE TITLE
401 - Unauthorized

### ISSUE THREAD

**Issue by MrHernandezTeaches** [floh22/LeagueBroadcastHub#153]
I've done several installs of this application. However, after working for a little while the page refreshes and I receive a 401 - Unauthorized

Exception type: EmbedIO.HttpException. 

Since this is the overlay being used as the browser source in OBS it will display the error instead of the data. Looking at the logs it appears that the events are being read and stored, but the display isn't being done on http://localhost:9001/frontend?backend=localhost because of the 401 error. I thought it could be my browser cache or conflicting extensions, but I refreshed/disabled all of it and it still didn't work.

**Comment by floh22** [comment]
can you open the overlay in your browser and open the console, then replicate this issue and post the console contents? This as well as LeagueBroadcast logs should help narrow down the issue

**Comment by MrHernandezTeaches** [comment]
All the logs end with "[WARNING] Could not save all configs"

I uploaded a few of the logs. Since I posted I did another fresh install, and more carefully modified the config files and ingame images. We're working fine for now. I cannot replicate the console in the browser anymore since it is functioning, but I dug through my search history and looked up this error from that day: Failed to Load Resource: net::ERR_BLOCKED_BY_CLIENT.

I'm going to close this issue, but shared everything if you wanted to take a glance. Thanks for making this for the LoL community!

[Crash-2022-05-21-00-01.log](https://github.com/floh22/LeagueBroadcast/files/8783375/Crash-2022-05-21-00-01.log)
[Log-2022-05-20-23-25.log](https://github.com/floh22/LeagueBroadcast/files/8783377/Log-2022-05-20-23-25.log)
[Log-2022-05-21-01-29.log](https://github.com/floh22/LeagueBroadcast/files/8783379/Log-2022-05-21-01-29.log)
"
